Dockerfile

```diff
@@ -1,4 +1,15 @@
-FROM rust:1.67-bullseye AS chef
+FROM rust:1.77-bullseye AS chef
```

Comment:
im not sure if this is worth the complexity but it would be great if we could just read the `rust-toolchain.toml` file to load the current MSRV

Reply:
I didn't found an approach to use rust version in Dockerfile, mb you can point me on some resources?
For now I added `lukemathwalker/cargo-chef:latest-rust-slim-bullseye` image